docs/public-api.asciidoc

```diff
@@ -82,7 +82,7 @@ var span = Elastic.Apm.Agent.Tracer.CurrentSpan;
 ==== `CaptureTransaction`
 
 This is a convenient method which starts and ends a transaction and captures unhandled exceptions.
-It has 3 parameters:
+It has 3 required parameters:
```

Comment:
One note: it's interesting that you separated required and optional parameters for `CaptureTransaction`, but for `CpatureSpan`, you grouped required and optional together (see https://github.com/elastic/apm-agent-dotnet/pull/811/files#diff-98d2de361a542d2ae87910714e3ddb27R508). For consistency, we should probably document the parameters in the same way for both methods

Reply:
Good catch, thanks. Changed `CaptureSpan`, so now they are separated for both methods.